src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java

```diff
@@ -1352,21 +1352,17 @@ during the analysis phase and thus cannot depends on a target result (the \
                     + " a path from.")
       })
   public StarlarkPath path(Object path) throws EvalException, InterruptedException {
-    return getPath("path()", path);
+    return getPath(path);
```

Comment:
probably just merge these two methods. if we ever need to un-merge, the starlark version can be renamed to 'pathForStarlark' or something.